docs/admin_guide.md

```
@@ -1291,42 +1291,42 @@ Track server activity with analytics dashboards and automated digest reports.
 
 **Enable analytics:**
 ```
-/insights setup action:enable
+/analytics setup action:enable
 ```
 
 **Disable analytics:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)